client/src/components/Header/Header.spec.js

```diff
@@ -0,0 +1,25 @@
+import { getByTestId, render, fireEvent } from "@testing-library/react";
```

Comment:
Remove unused imports
```suggestion
import {render } from "@testing-library/react";
```